cli/main.go

```diff
@@ -216,16 +216,6 @@ func main() {
 			appName+" stage [OPTIONS] SERVICE NEW_REVISION",
 			""),
 
-		builder.Command(&systemlogs.SystemLogs{},
-			"View system log for Rio management plane",
-			appName+" systemlogs",
-			""),
-
-		builder.Command(&systemconfig.SystemConfig{},
```

Comment:
Looks like we also drop rio systemlogs & rio systemconfig? Are we intending that `rio features` be the replacement for `rio systemlogs` in some way? Rio `systemlogs` is currently in our bug report template

Reply:
Looks like `rio systemlogs` got moved to `rio system logs`, but we will need to update bug report